### PR TITLE
Focus resume button when returning from playback and it's visible

### PR DESCRIPTION
**Pull request by Froghut**
I guess this is pretty self explanatory

### CODE REVIEW

**Comment by thornbill** [comment]
This may already be addressed in the redesign branch. I will defer to @nielsvanvelzen and @AndreasGB on that.

**Comment by nielsvanvelzen** [comment]
It's fine for now but it will be overwritten by the redesign branch

File: app/src/main/java/org/jellyfin/androidtv/details/FullDetailsActivity.java [hunk 1]
```diff
@@ -220,10 +220,15 @@ public void onResponse(BaseItemDto response) {
...
                                     mBaseItem = response;
                                     if (mResumeButton != null) {
-                                        mResumeButton.setVisibility((mBaseItem.getBaseItemType() == BaseItemType.Series && ! mBaseItem.getUserData().getPlayed()) || response.getCanResume() ? View.VISIBLE : View.GONE);
+                                        boolean resumeVisible = (mBaseItem.getBaseItemType() == BaseItemType.Series && !mBaseItem.getUserData().getPlayed()) || response.getCanResume();
+                                        mResumeButton.setVisibility(resumeVisible ? View.VISIBLE : View.GONE);
                                         if (response.getCanResume()){
                                             mResumeButton.setText(getString(R.string.lbl_resume_from, TimeUtils.formatMillis((response.getUserData().getPlaybackPositionTicks()/10000) - mApplication.getResumePreroll())));
                                         }
+                                        if (resumeVisible)
+                                        {
```
Comment (thornbill) [inline app/src/main/java/org/jellyfin/androidtv/details/FullDetailsActivity.java:229]: Super minor, but can you move the `{` to the previous line and include a space before it?

Comment (Froghut) [reply app/src/main/java/org/jellyfin/androidtv/details/FullDetailsActivity.java]: done

**Pull request merged**